applications/asset_tracker_v2/src/events/app_module_event.c

```diff
@@ -107,6 +107,7 @@ COMMON_EVENT_INFO_DEFINE(app_module_event,
 #endif /* CONFIG_PROFILER */
 
 COMMON_EVENT_TYPE_DEFINE(app_module_event,
-			 CONFIG_APP_EVENTS_LOG,
 			 log_event,
-			 &app_module_event_info);
+			 &app_module_event_info,
+			 IS_ENABLED(CONFIG_APP_EVENTS_LOG)
```

Comment:
I expect we should add flag inside `EVENT_FLAGS_CREATE` - it might be difficult to extend this snippet with new flags
Maybe w can use `IF_ENABLED` macro?

Let's check if following would work:
```
EVENT_FLAGS_CREATE(IF_ENABLED(CONFIG_APP_EVENTS_LOG), EVENT_TYPE_FLAGS_INIT_LOG_ENABLE),
                   IF_ENABLED(CONFIG_SOMETHING), EVENT_TYPE_FLAGS_SOMETHING_ENABLE))
```